app/src/main/java/nerd/tuxmobil/fahrplan/congress/settings/widgets/PreferenceListDialog.kt

```diff
@@ -0,0 +1,79 @@
+package nerd.tuxmobil.fahrplan.congress.settings.widgets
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.safeContentPadding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.foundation.selection.selectableGroup
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.DialogProperties
+import kotlinx.collections.immutable.ImmutableMap
+import nerd.tuxmobil.fahrplan.congress.designsystem.buttons.ButtonText
+import nerd.tuxmobil.fahrplan.congress.designsystem.buttons.RadioButton
+import nerd.tuxmobil.fahrplan.congress.designsystem.dialogs.AlertDialog
+import nerd.tuxmobil.fahrplan.congress.designsystem.modifiers.minimumInteractiveComponentSize
+import nerd.tuxmobil.fahrplan.congress.designsystem.texts.Text
+import nerd.tuxmobil.fahrplan.congress.designsystem.themes.EventFahrplanTheme
+
+@Composable
+internal fun <T> PreferenceListDialog(
+    title: String,
+    entries: ImmutableMap<T, String>,
+    selectedOption: T,
+    onOptionSelected: (T) -> Unit,
+    onDismiss: () -> Unit,
+) {
+    AlertDialog(
+        title = { Text(title) },
```

Comment:
The title appears pretty large compared to other texts in the app and compared to the former dialog:

<img width="270" height="600" alt="dialog-old" src="https://github.com/user-attachments/assets/6e4bb97b-17f0-4dea-b0c9-e19008b42e83" /> <img width="270" height="600" alt="dialog-new" src="https://github.com/user-attachments/assets/a0f2c85d-4703-48d0-9b37-b504e706618d" />

Reply:
Should the cancel button text be uppercased?

Reply:
As far as I can tell all of this is like it should be in Material 3: https://m3.material.io/components/dialogs/overview